R/calculate_within_batch_ari.R

```diff
@@ -19,7 +19,7 @@
 #'
 #' @export
 calculate_within_batch_ari <- function(individual_sce_list,
-                                       pc_name,
+                                       pc_names,
                                        merged_sce,
                                        batch_column = "library_id",
```

Comment:
Allowing the clustering parameter to be set.
```suggestion
                                       batch_column = "library_id",
                                       BLUSPARAM = bluster::NNGraphParam(cluster.fun = "louvain", type = "jaccard")
```